ts/main.ts

```diff
@@ -34,9 +34,102 @@ $formElement.addEventListener('submit', (event: Event) => {
     notes: $formElements.notes.value,
     entryId: data.nextEntryId,
   };
+
   data.nextEntryId++;
   data.entries.unshift(newEntry);
   writeData();
+
+  const li = renderEntry(newEntry);
```

Comment:
I would put a `$`before the variable name since it's storing a DOM element (the `li`)